rfcs/text/0040-volume-device.md

```diff
@@ -106,6 +106,7 @@ The following are the people that consulted on the contents of this RFC.
 
  * @Trinity2019    | author
  * @ricardoelastic | reviewer
```

Comment:
Should either @ricardoelastic or @stanek-michal be added as reviewers?

Reply:
Yes. I'll ask them in slack as well